.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Suggestion: Python package dependabot updates. I see there's a few possible updates, e.g. NumPy 1.19.5 -> 1.24.0.
```suggestion
version: 2
updates:

  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "monthly"

  - package-ecosystem: "pip"
    directory: "/packages/stats"
    schedule:
      interval: "monthly"
```